### DISCUSSION TITLE
How to get a reason why entry was not posted/added for unique fields?

### DISCUSSION THREAD

**Discussion by podeig** [strapi/strapi#6938] [Legacy Topics]
Hi,

I have an unique field in a collection "email". When I post a new entry I get just generic error without reason.
I wish to show to user message like "Your email was not added because it is already registered!".
How to do this? 

I receive just
```
{
  "statusCode": 500,
  "error": "Internal Server Error",
  "message": "An internal server error occurred"
}
```
![image](https://user-images.githubusercontent.com/32296171/86907061-c3ff0c00-c114-11ea-9c88-e292c9ab4750.png)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by jacargentina** [comment]
I need an answer to this too (https://github.com/strapi/strapi/issues/5863)
